### DISCUSSION TITLE
Idea - Lockscreen?

### DISCUSSION THREAD

**Discussion by horlufemi** [t1m0thyj/WinDynamicDesktop#314] [Ideas]
I really like your app. Great job Tim!

I have an idea. can you make the wallpaper sync with the lock screen? i.e. make the wallpaper show on the lockscreen also.

Thank you.

Regards

**Comment by sugar-king** [comment]
There is an experimental script available for WDD that can change lockscreen wallpaper: https://windd.info/scripts/
However, due to some Windows bug, it does not work consistently. 

Also see #257.

**Reply by t1m0thyj** [reply]
Changing the lock screen image has been added in WDD 5.0 and should now work smoothly.
